engine/src/dskw32.cpp

```diff
@@ -3697,6 +3755,13 @@ struct MCWindowsDesktop: public MCSystemInterface, public MCWindowsSystemService
 			// absolute path
 			t_canonised_path = p_path;
 		}
+		
+		// PM-2016-03-15: [[ Bug 16300 ]] Detect correctly the case of a serial port path
+		else if (isValidSerialPortPath(MCStringRef p_path))
+		{
+			t_canonised_path = MCValueRetain(p_path);
```

Comment:
This shouldn't be `MCValueRetain`ed - something assigned to an AutoValueRef is automatically retained.